src/validate_pyproject/formats.py

```diff
@@ -378,7 +378,25 @@ def int(value: builtins.int) -> bool:
     return -(2**63) <= value < 2**63
 
 
-def SPDX(value: str) -> bool:
-    """Should validate eventually"""
-    # TODO: validate conditional to the presence of (the right version) of packaging
-    return True
+try:
```

Comment:
Wouldn't it make sense to make a single function that tries the import inside, rather than making one of two alternate functions? This just seems like a rather odd design. The import is cached so you won't even lose significant time checking it on each call.

Reply:
I think that the main point of for the warning to just be issued once.

Reply:
Don't warnings only show up once by default?

Reply:
Okay, fine to go with this, then.